src/payload/checkpoint.rs

```diff
@@ -475,6 +489,11 @@ impl<P: Platform> Debug for Checkpoint<P> {
 
 impl<P: Platform> Display for Checkpoint<P> {
 	fn fmt(&self, f: &mut core::fmt::Formatter<'_>) -> core::fmt::Result {
+		let tag_suffix = self
```

Comment:
Nit: can we move this into the function? Would it make sense? 
To call something like self.tag_display()